Portfolio form section branch

forLoop94 opened the pull request:
1. Name field accepts up to 30 chars.
2. Email field accepts only emails.
3. Message field accepts up to 500 chars.
4. All fields have placeholder text.
5. All fields have the correct type.
6. A user cannot submit the form if one or more of the fields is empty.
7. I Collect data by using the [Formspree service](https://formspree.io/html).
8. I used Flexbox to lay out elements on the page.
9. I followed the design as much as possible, using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
10. I Implemented the button interactions.

tufoinnkuo10 reviewed (changes requested):
Hi @forLoop94  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a live demo section that does not work. please kindly remove it. you will add it when your project is deployed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

forLoop94 commented:
1. Name field accepts up to 30 chars.
2. Email field accepts only emails.
3. Message field accepts up to 500 chars.
4. All fields have placeholder text.
5. All fields have the correct type.
6. A user cannot submit the form if one or more of the fields is empty.
7. I Collect data by using the [Formspree service](https://formspree.io/html).
8. I used Flexbox to lay out elements on the page.
9. I followed the design as much as possible, using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
10. I Implemented the button interactions.
11. I removed the live demo from the README file

NahnahAJ commented on index.html:
```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
+
+    <!--  contact form section -->
+    <!-- added by habtamualemayehu16@gmail.com   -->
+    <section class="reg-form">
+      <div class="form-body">
+        <div class="header-text">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
```
- Great job so far @forLoop94 your form works great. There are just a few style adjustments you need to make. You can see from the image below that your contact session is missing some padding as compared to the Figma design. There isn't enough spacing between the `Skills section` and the `Contact` section so please fix this so your design looks like the Figma design :100: 
- [ ]  You also need to increase the width of the contact page header text. It has a smaller width compared to the Figma Design :100: 

| Your Design | The Figma Design |
|--------|--------|
| ![Screenshot from 2023-03-29 15-18-17](https://user-images.githubusercontent.com/102699958/228591673-1bd3aba3-cf95-403b-9042-cb424a4512d8.png) | ![Screenshot from 2023-03-29 15-24-37](https://user-images.githubusercontent.com/102699958/228591249-c4c1ed4d-bd29-476a-8821-c564e6bfb086.png) |

NahnahAJ commented on index.html:
```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
+        <div class="header-text">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        </div>
+        <form action="https://formspree.io/f/xknaqwwd" method="post">
+          <input type="text" class="text-field" placeholder="Full name" min="1" max="30" name="Name" required>
+          <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+          <textarea name="Bio" id="bio" class="text-field" cols="30" rows="5" min="1" max="500" placeholder="Write me something"></textarea required>
```
- [ ] I noticed even though your  `textarea` has the `required` keyword,  it is not being used in the right place. As a result, the form was submitted even if it wasn't filled. Kindly fix this to ensure that your `Form` is properly validated :+1: 
- [ ] I also noticed that you didn't use the right keyword for ensuring that these requirements are met. You should use the `maxlength` attribute and not `max` as you've used currently :+1: So kindly fix these as well and ensure that the below requirements are met :+1: 
```
Name field accepts up to 30 chars.
Message field accepts up to 500 chars.
```

NahnahAJ commented on index.html:
```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
+          <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+          <textarea name="Bio" id="bio" class="text-field" cols="30" rows="5" min="1" max="500" placeholder="Write me something"></textarea required>
+          <div class="form-btn">
+              <button type="submit" class="about-me-btn">Get in touch</button>
```
- [ ] Your button currently doesn't have the right button interaction styles as shown in the Figma design. So kindly fix this as well :100: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/228594977-62417279-f315-4b43-b4fe-97c93e08b817.png)

NahnahAJ commented on index.html:
```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
+          <textarea name="Bio" id="bio" class="text-field" cols="30" rows="5" min="1" max="500" placeholder="Write me something"></textarea required>
+          <div class="form-btn">
+              <button type="submit" class="about-me-btn">Get in touch</button>
+          </div>
+        </form>
+      </div>
+    </section>
```
- [ ] Lastly, please ensure that the spacing between the `button` and the `footer` is similar to that of the Figma Design :+1: 

| The Figma Design |  Your Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/228595646-56069a56-6de4-4c5c-9531-ac80d9f4b225.png) | ![image](https://user-images.githubusercontent.com/102699958/228595833-6f0fa725-6370-4844-abe6-678c5bea2758.png) |

NahnahAJ commented on ReadMe.md:
```diff
@@ -52,13 +52,6 @@
 - **Use professional documentation**
 - **Use HTML and CSS**
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [Live Demo Link](https://google.com)
```
- [ ] I will suggest that you include this section and just put `N/A` instead of including a false link. :+1: eg.

```
## 🚀 Live Demo <a name="live-demo"></a>

- N/A
```

NahnahAJ reviewed (changes requested):
## Project Status: CHANGES REQUIRED :recycle: 
Hi @forLoop94 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media2.giphy.com/media/m3864rBwwBTKMAbpn3/giphy.gif"/>

## Highlights
- No linter errors :heavy_check_mark: 
- Nice Design :heavy_check_mark: 
- Form well set up :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

huseyinbiyik commented on styles.css:
```diff
@@ -230,13 +230,16 @@ h2 {
+  margin-bottom: 60px;
 }
 
 .btn-type-a:hover,
```
Great job by adding hover state to submit button. But as the previous reviewer requested, we need to consider all the states of button. Here you have missing "pressed state". Please stick to the color on Figma template. You can implement with :active selector. These states are useful to increase the user experience of your web page.

huseyinbiyik reviewed (changes requested):
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![tenor](https://user-images.githubusercontent.com/25887253/205415161-00e79d1b-af68-4f10-9b2c-265f460dc5da.gif)

 ## Highlights
- The contact form works very well 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Please always capitalize the first word of your commit messages. It will increase the readability of your commit messages.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

forLoop94 commented:
1. I have added the "pressed state" for the buttons using :active. I have done this following the Figma template.

Thank you.

mahbubul14 reviewed (approved):
# Project Approved :trophy: :tada: 

Hi @forLoop94, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
